metis-media-service/src/main/java/eu/europeana/metis/mediaprocessing/extraction/TextProcessor.java

```diff
@@ -107,6 +118,26 @@ public ResourceExtractionResultImpl extractMetadata(Resource resource, String de
     return new ResourceExtractionResultImpl(metadata, thumbnails);
   }
 
+  private static void closeAllThumbnailsSilently(List<Thumbnail> thumbnails) {
+    if (thumbnails != null) {
+      for (Thumbnail thumbnail : thumbnails) {
+        try {
+          thumbnail.close();
+        } catch (IOException e) {
+          LOGGER.warn("Could not close thumbnail: {}", thumbnail.getTargetName(), e);
+        }
+      }
+    }
+  }
+
+  void removePdfImageFileSilently(Path file) {
```

Comment:
Isn't this duplicate, same code exists in the PdfToImageConverter

Reply:
Fixed.